parallaxinc/Parallax-IDE
# Console logger moved into flux

#### What's this PR do?

Brings the debug console actions into Alt Flux.
#### Where should the reviewer start?

Look at the new action and store for console.js.
The download event now triggers an action to output to the debug console.
#### How should this be manually tested?

Ensure download and logging to the console are still working. Test that frame rate does not bog down your computer with an output than generates hundreds of lines in a loop.
#### Any background context you want to provide?

Overall request to move ChromeIDE to flux.
#### What are the relevant tickets?

References #115
